### ISSUE TITLE
Crash with SOCIAL_AUTH_SAML - signal 11

### ISSUE THREAD

**Issue by thoro** [netbox-community/netbox-docker#1146]
### Current Behavior

When returning from the SAML Idp to the url /oauth/complete/saml/ netbox app crashes with signal 11.

It shows Error 503 on the website.

Reloading the page allows for the login to continue, but restarting the Login always leads to the 503 error.

### Expected Behavior

Correctly login and not show an error

### Docker Compose Version

kubernetes

### Docker Version

```text
Image Builder:

Client:
 Cloud integration: v1.0.29
 Version:           20.10.17
 API version:       1.41
 Go version:        go1.17.11
 Git commit:        100c701
 Built:             Mon Jun  6 23:04:45 2022
 OS/Arch:           darwin/arm64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.12.0 (85629)
 Engine:
  Version:          20.10.17
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.11
  Git commit:       a89b842
  Built:            Mon Jun  6 23:01:01 2022
  OS/Arch:          linux/arm64
  Experimental:     true
 containerd:
  Version:          1.6.8
  GitCommit:        9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

Kubernetes is running containerd://1.6.24
```

### The git Revision

2.8.0

### The git Status

```text
HEAD detached at 2.8.0
nothing to commit, working tree clean
```

### Startup Command

default image startup

### NetBox Logs

```text
127.0.0.1 - - [01/Feb/2024:13:11:02 +0000] "GET /oauth/login/saml/?next=%2F&idp=xxxx HTTP/1.0" 302 0 "https://xxx.xxx.xxx/login/?next=/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
2024/02/01 13:11:02 [alert] 19#19 app process 20 exited on signal 11
127.0.0.1 - - [01/Feb/2024:13:11:02 +0000] "POST /oauth/complete/saml/ HTTP/1.0" 503 54 "https://yyy.yyy.yyy/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
```

### Content of docker-compose.override.yml

```yaml
don't have that.
```

**Comment by jonlarsdizparc** [comment]
Experiencing the same issue myself. Using SAML towards Azure if that helps.

**Comment by ghost** [comment]
I am also experiencing this on a 3.7.2 deployment on K8s using the chart. It seems to be related to this: 
https://github.com/netbox-community/netbox/discussions/14649

Downgrading to 3.6.7 causes the issue to go away. 
I'm not sure there's an easy way to fix this from the operators perspective without a change to the container.

**Comment by tobiasge** [comment]
Fixed with #1135.
